CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Change Log
 
+## 0.17.0 - 2020-05-26
+### Added
+- Enhanced deployment logs to show more information and more accurately detect failures
+- Added critical insight notification to help diagnose issues when a deploy succeeds, but the app fails to start
```

Comment:
🙁 I see how it is...

Reply:
![1](https://i.makeagif.com/media/9-30-2015/EJVWKv.gif)